examples/island_cluster_with_vm_router/main.tf

```diff
@@ -0,0 +1,219 @@
+resource "google_service_account" "gke-sa" {
```

Comment:
Any downside to using the SA created by the module?

Reply:
This is required for this specific example, since the SA is used for PSC and proxy firewall rules created ahead of the cluster.